doc/developer-getting-started.md

```diff
@@ -25,7 +25,8 @@ At this point, the website will be live on http://localhost:8000 but there is no
 data in the database. The ``initialise-developer-site.sh`` script performs an
 initial database migration and installs some test data in the site. Two users
 are created, ``test0001`` and ``test0002``. The ``test0001`` user is configured
-as a superuser. The script can be run inside the webapp container:
+as a superuser. A pre-allocated Site and associated entities is also created. 
```

Comment:
Maybe, <tt>:py:mod:\`~sitesmanagement.models.Site\`</tt>?

Reply:
http://www.sphinx-doc.org/en/stable/domains.html#the-python-domain

Reply:
... and http://www.sphinx-doc.org/en/stable/domains.html#cross-referencing-python-objects